lib/experimental/Navigation/Sidebar/CompanySelector/index.tsx

```diff
@@ -40,10 +51,13 @@ export function CompanySelector({
     )
   }
 
-  if (companies.length === 1) {
+  if (companies.length + (additionalOptions?.length || 0) === 1) {
```

Comment:
what do you think about setting `additionalOptions` default value to `[]`? it will eliminate all extra checks